docs/building_the_project_with_dpcpp.rst

```diff
@@ -3,32 +3,31 @@
 Building the Project with DPC++
 ===============================
 
-This page describes building the oneMKL Interfaces with either the Intel(R)
-oneAPI DPC++ Compiler or open-source oneAPI DPC++ Compiler. For guidance on
-building the project with AdaptiveCpp, see
-:ref:`building_the_project_with_adaptivecpp`.
+This page describes building the oneMath with either the Intel(R) oneAPI DPC++
```

Comment:
```suggestion
This page describes building oneMath with either the Intel(R) oneAPI DPC++
```

Reply:
Thanks, that's fixed in https://github.com/oneapi-src/oneMKL/pull/602/commits/463fe574ce1e90d3314078b12551f0529c637167
It doesn't look like there were any other occurrences like this.